src/dev/build/tasks/os_packages/docker_generator/templates/dockerfile.template.js

```diff
@@ -120,7 +133,7 @@ function generator({
     org.label-schema.build-date="${dockerBuildDate}" \\
     license="${license}"
 
-  ENTRYPOINT ["/usr/local/bin/dumb-init", "--"]
+  ENTRYPOINT ["/tini", "--", "/usr/local/bin/dumb-init", "--"]
```

Comment:
is there overlap between tini and dumb-init?  should we remove dumb-init?

Reply:
Yeah, totally spaced on that.